network/arp_packet.ksy

```diff
@@ -0,0 +1,28 @@
+meta:
+  id: arp_packet
+  title: ARP network packet
+  endian: be
+seq:
+  - id: hardware_type
+    type: u2be
+  - id: protocol_type
+    type: u2
+  - id: hardware_size
+    type: u1
```

Comment:
Please consider using `len_hardware` and similar identifiers to be aligned with [KSY style guide](https://doc.kaitai.io/ksy_style_guide.html)